src/index.js

```diff
@@ -59,5 +92,6 @@ const css = (...styleDefinitions) => {
 export default {
```

Comment:
Thanks for the headsup. It's been discussed ab it before, and if/when we do decide to upgrade to Babel 6, we have this PR ready: https://github.com/Khan/aphrodite/pull/27 (which also removes this `default`).

If removing that `default` is a non-breaking change, I'm also happy to take a PR with just that.